src/sensor/sensor_supervisor.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+from sensor import sensor
+
+
+class sensorSupervisor:
```

Comment:
Rewrite class name to UpperCamelCase
```suggestion
class SensorSupervisor:
```